js/widgets/toolbar.js

```diff
@@ -47,7 +48,7 @@ return $.widget( "mobile.toolbar", {
 
 	_create: function() {
 		var leftbutton, rightbutton,
-			role = this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
+			role =  this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
```

Comment:
we need to make this an option with the ability to set a string for things like `role="toolbar"` actually that should probably be the default. we cant use role as an option it will conflict with the init selector. maybe contentRole? open to suggestions

/cc @gabrielschulhof  @jaspermdegroot